tiledb/rest/capnp/query.cc

```diff
@@ -107,8 +170,7 @@ tiledb::sm::Status query_deserialize(
             message_builder.initRoot<rest::capnp::Query>();
         json.decode(kj::StringPtr(serialized_string), query_builder);
         rest::capnp::Query::Reader query_reader = query_builder.asReader();
-        return query->from_capnp(&query_reader);
-        break;
+        return query->from_capnp(&query_reader, nullptr);
```

Comment:
We need to support buffer serialization with json too, it is okay to still ship the raw buffers at the end of the json string. I think for json though you'll have to manually split the string vs the buffers, I am not certain captain proto will parse it correctly, like it does capnp format.